edge/services/helloworld/PolicyRegister.md

```diff
@@ -101,46 +74,35 @@ hzn policy list
 
 - Note this simple Service Policy doesn't provide any `properties`, but it does have a `constraint`. This example `constraint` is one that a Service developer might add, stating that their Service must only run on the models named `Whatsit ULTRA` or `Thingamajig ULTRA`. If you recall the Node Policy we used above, the model `property` was set to `Thingamajig ULTRA`, so this Service should be compatible with our Edge Node.
 
-1. List the services in your org:
-```bash
-hzn exchange service list
-```
-
-2. To attach the example Service policy to this service, use the following command (substituting your service name):
-
-```bash
-hzn exchange service addpolicy -f horizon/service_policy.json <published-helloworld-service-name>
-```
-
-3. Once that completes, you can look at the results with the following command:
+1. View the pubished service policy attached to `ibm.helloworld`:
 
 ```bash
-hzn exchange service listpolicy <published-helloworld-service-name>
+hzn exchange service listpolicy IBM/ibm.helloworld_1.0.0_amd64
 ```
 
 - Notice that Horizon has again automatically added some additional `properties` to your Policy. These generated property values can be used in `constraints` in Node Policies and Business Policies.
 
-- Now that we have set up the Policies for an Edge Node and the Policies for a published Service, we can move on to the final step of defining a Business Policy to tie them all together and cause software to be automatically deployed on your Edge Node.
+- Now that you have set up the Policy for your Edge Node and the published Service policy is in the exchange, we can move on to the final step of defining a Business Policy to tie them all together and cause software to be automatically deployed on your Edge Node.
 
 ### Business Policy
 
 - Business Policy (sometimes called Deployment Policy) is what ties together Edge Nodes, Published Services, and the Policies defined for each of those, making it roughly analogous to the Deployment Patterns you have previously worked with.
 
 - Business Policy, like the other two Policy types, contains a set of `properties` and a set of `constraints`, but it contains other things as well. For example, it explicitly identifies the Service it will cause to be deployed onto Edge Nodes if negotiation is successful, in addition to configuration variable values, performing the equivalent function to the `-f horizon/userinput.json` clause of a Deployment Pattern `hzn register ...` command. The Business Policy approach for configuration values is more powerful because this operation can be performed centrally (no need to connect directly to the Edge Node).
 
-- Below is the `horizon/business_policy.json` file used for this example:
+- Below is the `business_policy.json` file you obtained in step one:
 
 ```json
 {
-  "label": "$SERVICE_NAME Business Policy for $ARCH",
+  "label": "ibm.helloworld Business Policy for $ARCH",
```

Comment:
Some comment as in the cpu2eventstreams PR (get hzn.json and eval it)

Reply:
done